tests/test_vector2_rotate.py

```diff
@@ -88,26 +96,27 @@ def test_rotation_stability2(initial, angles):
 
 
 @given(
-    a=vectors(max_magnitude=1e150), b=vectors(),
+    x=vectors(max_magnitude=1e150), y=vectors(),
     l=st.floats(min_value=-1e150, max_value=1e150),
     angle=st.floats(min_value=-360, max_value=360),
 )
 # In this example:
-# * a * l == -b
+# * x * l == -y
 # * Rotation must not be an multiple of 90deg
 # * Must be sufficiently large
 @example(
-    a=Vector2(1e10, 1e10),
-    b=Vector2(1e19, 1e19),
+    x=Vector2(1e10, 1e10),
+    y=Vector2(1e19, 1e19),
     l=-1e9,
     angle=45,
 )
-def test_rotation_linearity(a, b, l, angle):
-    inner = (l * a + b).rotate(angle)
-    outer = l * a.rotate(angle) + b.rotate(angle)
-    note(f"l * a + b: {l * a + b}")
-    note(f"l * a.rotate(): {l * a.rotate(angle)}")
-    note(f"b.rotate(): {b.rotate(angle)}")
+def test_rotation_linearity(x, y, l, angle):
+    """(l*x + y).rotate is equivalent to l*x.rotate + y.rotate"""
```

Comment:
This helps, I feel like we could actually name `l` as something. Even as simple as `scalar` might be enough.